C++/C++.sublime-syntax

```diff
@@ -1268,6 +1292,7 @@ contexts:
     - match: '(?=\{)'
       set: function-definition-body
     - include: function-specifiers
+    - include: constraint-requires
```

Comment:
As `constraint-requires` sets a context on stack, the pattern on line 1292 will never match in function concepts, thus body is not scoped `meta.function meta.block`.

Reply:
#[b58a74c](https://github.com/sublimehq/Packages/pull/4140/commits/b58a74c2ba1dc5a6700728e939d64cb5dfe97152)